src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -438,13 +506,12 @@ export function useSwapInputsController({
   };
 
   const quoteFetchingInterval = useAnimatedInterval({
-    intervalMs: 10_000,
+    intervalMs: 12_000,
```

Comment:
should extract this value out and name is like ESTIMATED_BLOCK_TIME or something